templates/OperationResponseExceptionSnippet.stencil

```diff
@@ -1,14 +1,20 @@
 {% for exception in op.exceptions %}
-    {% if exception.hasDefaultException == false %}
```

Comment:
Is this a regression? What if there is no defaultException?

Reply:
this is not a regression. this is a necessary if check in the stencil. Because we already filter out default Exception in  op.exceptions when we build the view model. i.e. all exceptions in op.op.exceptions will be hasDefaultException equals to false. No point to have another check